src/main/java/com/kodilla/ecommercee/domain/dto/CartDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class CartDto {
+
+    private Long Id;
```

Comment:
I don't know quays but according to specs we don't have id for cart here, just list of products.

Reply:
Mine CartDto will have id

Reply:
Our Class Cart will have id. According to ours specs CartDto might not need it. I'm not sure about it though and I'm not gonna hold to it. Just suggestion.